modules/msk/genomenexus/vcf2maf/meta.yml

```diff
@@ -0,0 +1,52 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "genomenexus_vcf2maf"
+
+description: DSL2 module to perform a native conversion of VCF file type to MAF file type using the genomenexus suite of conversion and annotation tools.
+keywords:
+  - convert
+  - vcf
+  - maf
+  - genomenexus
+  - genomics
+tools:
+  - "genomenexus":
+      description: "DSL2 module to perform a native conversion of VCF file type to MAF file type using the genomenexus suite of conversion and annotation tools."
+      homepage: "https://github.com/genome-nexus"
+      documentation: "https://github.com/genome-nexus/.github/blob/main/profile/README.md"
+
+input:
+  # Only when we have meta
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'test', case_id:'tumor', control_id:'normal' ]`
```

Comment:
You mention `case_id` and `control_id` here, but I don't see them used in `main.nf`? Do we need to add some logic to process the ids? Maybe append them to output or the command?